scripts/audit_frontend_licenses.py

```diff
@@ -169,15 +144,15 @@ def check_licenses(licenses) -> NoReturn:
         print(f"{len(bad_packages)} unacceptable licenses")
         sys.exit(1)
 
-    print(f"No unacceptable licenses")
+    print(f"No unacceptable licenses: {len(packages)} checked")
     sys.exit(0)
 
 
 def main() -> NoReturn:
     # Run `yarn licenses` for lib.
     licenses_output = (
         subprocess.check_output(
-            ["yarn", "licenses", "list", "--json", "--production", "--ignore-platform"],
+            ["yarn", "licenses", "list", "--json", "--production", "--recursive"],
```

Comment:
nit: what was the reason for using `ignore-platform` before? And is this something we don't need anymore with the upgrade?

Reply:
The new plugin does not support ignore-platform, and this flag was mainly for yarn install, so it was unnecessary for us.